seventeenth_hw/tests/test_manager_application_form.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+
+from seventeenth_hw.utilities.config_reader import get_manager_application_form_name_key, \
+    get_manager_application_form_phone_key, get_manager_application_form_text_key
+
+
+@pytest.mark.regression
+def test_exist_phone_onfocus(open_main_page):
+    main_page = open_main_page
+    form = main_page.click_search_submit().click_manager_application_button()
+    assert form.is_phone_onfocus_exist(), "Manager application form onfocus is not displayed"
+
+
+@pytest.mark.smoke
+def test_close_form(open_main_page):
+    main_page = open_main_page
+    form = main_page.click_search_submit().click_manager_application_button()
```

Comment:
you have this line in all test of this suite. So why don't just create fixture with this code